provisioning/roles/php/tasks/main.yml

```diff
@@ -1,25 +1,26 @@
 ---
 - name: (purge HHVM)
   apt: name=hhvm state=absent
+  
+- name: add repository
+  apt_repository: repo='ppa:ondrej/php'
```

Comment:
I think this would not update the package list, you can consider adding an `apt: update_cache=true` action after. or adding the repository before the packages are updated in the first place.